packages/beacon-node/test/perf/chain/verifyImportBlocks.test.ts

```diff
@@ -7,11 +7,14 @@ import {defaultOptions as defaultValidatorOptions} from "@lodestar/validator";
 // eslint-disable-next-line import/no-relative-packages
 import {rangeSyncTest} from "../../../../state-transition/test/perf/params.js";
 import {
-  beforeValue,
   getNetworkCachedState,
   getNetworkCachedBlock,
   // eslint-disable-next-line import/no-relative-packages
-} from "../../../../state-transition/test/utils/index.js";
+} from "../../../../state-transition/test/utils/testFileCache.js";
```

Comment:
Since we have @lodestar/test-utils package now, does it make sense to move all of these utils from state-transtion there?

Reply:
Yes it definitely make sense, but will do it as separate PR.